src/main/java/frc/robot/RobotContainer.java

```diff
@@ -209,22 +249,43 @@ private void configureButtonBindings() {
     }
 
     if (intakeSubsystem != null && centererSubsystem != null && indexerSubsystem != null) {
-      intakeButton.whileHeld(new IntakeCommand(intakeSubsystem, centererSubsystem, indexerSubsystem));
-      intakeDeploy.whenPressed(new InstantCommand(() -> intakeSubsystem.deploy()));
-      intakeRetract.whenPressed(new InstantCommand(() -> intakeSubsystem.retract()));
+      driverIntakeButton.whileHeld(new IntakeCommand(intakeSubsystem, centererSubsystem, indexerSubsystem));
+      operatorIntakeButton.whenPressed(new IntakeCommand(intakeSubsystem, centererSubsystem, indexerSubsystem));
```

Comment:
I think the operator button will also be while held

Reply:
fixed